Frameworks/CoreGraphics/CGGradient.mm

```diff
@@ -153,7 +152,7 @@ CGGradientRef CGGradientCreateWithColorComponents(CGColorSpaceRef colorSpace,
  @Status Interoperable
 */
 CGGradientRef CGGradientCreateWithColors(CGColorSpaceRef colorSpace, CFArrayRef colors, const CGFloat* locations) {
-    RETURN_NULL_IF(!colorSpace);
+    // colorspace can be null
     RETURN_NULL_IF(!colors);
     RETURN_NULL_IF(!__isValidGradientColorspaceModel(colorSpace));
```

Comment:
@DHowett-MSFT actually CGColorSpaceGetModel  returns unknown.
It's really hard to know what's happening and how the colors are being treated. I'm attempting some tests on ref platform to figure out how the gradients are being rendered.